hibernate-community-dialects/src/main/java/org/hibernate/community/dialect/IngresDialect.java

```diff
@@ -144,6 +143,30 @@ public IngresDialect(DatabaseVersion version) {
 			registerColumnType( Types.DATE, "ansidate" );
 		}
 
+		limitHandler = getVersion().isBefore( 9, 3 ) ? FirstLimitHandler.INSTANCE : IngresLimitHandler.INSTANCE;
+
+		sequenceSupport = new ANSISequenceSupport() {
+			@Override
+			public boolean supportsPooledSequences() {
+				return getVersion().isSameOrAfter( 9, 3 );
+			}
+		};
+	}
+
+//	@Override
+//	protected void initDefaultProperties() {
+//		super.initDefaultProperties();
+//
+//		if ( getVersion().isBefore( 10 ) ) {
+//			// There is no support for a native boolean type that accepts values
+//			// of true, false or unknown. Using the tinyint type requires
+//			// substitutions of true and false.
+//			getDefaultProperties().setProperty( Environment.QUERY_SUBSTITUTIONS, "true=1,false=0" );
+//		}
+//	}
```

Comment:
Why was this not commented back in? Not needed anymore or was that by accident?

Reply:
Well it *shouldnt* be necessary, as far as I can see. But of course I have no way on earth to test that, so I left the commented code there just in case someone comes back and tells us removing it broke something.

Reply:
Right, it's not even used anymore AFAICS. How about removing the property altogether?

Reply:
Yes, we should. That one was already on my list.